src/components/Counter.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import './Counter.scss';
+
+type State = {
+  count: number;
+};
+
+export class Counter extends React.Component<{}, State> {
+  state = {
+    count: 0,
+  };
+
+  addOne = () => {
+    this.setState((state) => ({
+      count: state.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState((state) => ({
+      count: state.count + 100,
+    }));
+  };
+
+  increase = () => {
+    if (this.state.count % 5 === 0) {
+      this.add100();
+    }
+
+    this.addOne();
+  };
+
+  render() {
+    const { count } = this.state;
+
+    return (
+      <div className="
+        has-background-dark
+        main-container"
+      >
+        <h1 className="title is-1 has-text-warning">
+          {`Count: ${count}`}
+        </h1>
+
+        <button
+          type="button"
+          className="button
+            is-success
+            is-large
+            is-rounded"
+          onClick={() => {
+            this.addOne();
+          }}
```

Comment:
```suggestion
          onClick={this.addOne}
```
you can remove the arrow function because method addOne doesn't accept  any arguments